### DISCUSSION TITLE
Protecting the Wiki

### DISCUSSION THREAD

**Discussion by nouwaarom** [clibs/clib#250] [Q&A] [answered]
I noticed that the wiki is currently editable  for everyone. While this was very useful when working on clib, I think that it is better to protect it to make sure nobody breaks the package listing.

**Comment by stephenmathieson** [comment, marked as answer]
This has been brought up several times over the many years this project has been alive. I personally like that it's open and believe it lowers the barrier to contributing new packages. Furthermore, changes are easily reverted (the wiki is a Git repo), so if a malicious change was made, we could ban the user and reset the wiki.

My stance here is to not fix what isn't broken. If it breaks in the future, we'll address it then.

Curious to hear what others think, as it's been awhile since we've talked about this.

**Reply by jwerle** [reply]
I agree with @stephenmathieson. We can address this when it becomes an issue. Perhaps in the future, clibs.org can support Github accounts and a means for publishing there (possibly modifying the wiki)
